llm/server.go

```diff
@@ -204,6 +204,11 @@ func NewLlamaServer(gpus gpu.GpuInfoList, model string, ggml *GGML, adapters, pr
 		params = append(params, "--numa")
 	}
 
+	flashAttnSupported := (gpus[0].Library == "cuda" && gpus[0].Major >= 7 || gpus[0].Library == "metal")
```

Comment:
For a multi-GPU setup, wouldn't we need to ensure all the cuda cards are major 7 or greater?

Reply:
Good catch! I totally would have missed that 🤦 , Updated.

Reply:
The gpus class had a change overnight, had to refactor.

Reply:
@dhiltgen are you happy with this refactor?